app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
just for reference -this was removed from ember-redux just before the 2.0 release

https://github.com/ember-redux/ember-redux/commit/be2f989cea73a802a0afaacb011dffa59cef4e09